Birthday-2024-Project/MainScenes/screen_mngr_init_scene.tscn

```diff
@@ -1,6 +1,34 @@
-[gd_scene load_steps=5 format=3 uid="uid://crnv5o8nb1u1k"]
+[gd_scene load_steps=36 format=3 uid="uid://crnv5o8nb1u1k"]
 
 [ext_resource type="Script" path="res://Scripts/ScreenManagement/ScreenManager.gd" id="1_karo4"]
+[ext_resource type="Texture2D" uid="uid://b63lb17d6eycv" path="res://Art/UI/PageTransition/PageTurnAnimation-01.png" id="2_01n30"]
+[ext_resource type="Texture2D" uid="uid://28d5485f4d46" path="res://Art/UI/PageTransition/PageTurnAnimation-02.png" id="3_7nwg4"]
+[ext_resource type="Texture2D" uid="uid://6qwnsvi0hkos" path="res://Art/UI/PageTransition/PageTurnAnimation-03.png" id="4_bra01"]
+[ext_resource type="Texture2D" uid="uid://d03op7ghvsh6t" path="res://Art/UI/PageTransition/PageTurnAnimMask-02.png" id="5_rhsnw"]
+[ext_resource type="Texture2D" uid="uid://c0lkpcmj42ncy" path="res://Art/UI/PageTransition/PageTurnAnimation-04.png" id="5_wk5ea"]
+[ext_resource type="Texture2D" uid="uid://4m0m8xkwdci8" path="res://Art/UI/PageTransition/PageTurnAnimation-05.png" id="6_v70ys"]
+[ext_resource type="Texture2D" uid="uid://de8ahix7bmhx7" path="res://Art/UI/PageTransition/PageTurnAnimation-06.png" id="7_x88me"]
+[ext_resource type="Texture2D" uid="uid://5j76ybjxidrj" path="res://Art/UI/PageTransition/PageTurnAnimation-07.png" id="8_iuns0"]
+[ext_resource type="Texture2D" uid="uid://blmi1vq3211tr" path="res://Art/UI/PageTransition/PageTurnAnimation-08.png" id="9_djcfe"]
+[ext_resource type="Texture2D" uid="uid://plept3uyf7pg" path="res://Art/UI/PageTransition/PageTurnAnimation-09.png" id="10_dp4gw"]
+[ext_resource type="Texture2D" uid="uid://huwdudx8oe6e" path="res://Art/UI/PageTransition/PageTurnAnimMask-00.png" id="11_w8gcn"]
+[ext_resource type="Texture2D" uid="uid://df1eo3kycls1j" path="res://Art/UI/PageTransition/PageTurnAnimMask-01.png" id="12_hf7di"]
+[ext_resource type="Texture2D" uid="uid://byn2h2431otn4" path="res://Art/UI/PageTransition/PageTurnAnimMask-03.png" id="14_g15bj"]
+[ext_resource type="Texture2D" uid="uid://dudvvx8i3ni1k" path="res://Art/UI/PageTransition/PageTurnAnimMask-04.png" id="15_wgwj2"]
+[ext_resource type="Texture2D" uid="uid://dlbnctqoyvf68" path="res://Art/UI/PageTransition/PageTurnAnimMask-05.png" id="16_hjmb6"]
+[ext_resource type="Texture2D" uid="uid://cllelhg07avo5" path="res://Art/UI/PageTransition/PageTurnAnimMask-06.png" id="17_o3m4r"]
+[ext_resource type="Texture2D" uid="uid://drw0nv1dpx86p" path="res://Art/UI/PageTransition/PageTurnAnimMask-07.png" id="18_fmpkw"]
+[ext_resource type="Texture2D" uid="uid://cpwl0mab8uapu" path="res://Art/UI/PageTransition/PageTurnAnimMask-08.png" id="19_ksfsp"]
+[ext_resource type="Texture2D" uid="uid://b25127ub1esc" path="res://Art/UI/PageTransition/PageTurnAnimMaskInverted-08.png" id="20_n218r"]
+[ext_resource type="AudioStream" uid="uid://dbam5qe2kcrfx" path="res://Resources/Audio/SFX/PageTurnBack.wav" id="20_ogexp"]
```

Comment:
is it just me or are PageTurnBack and Next being played at random when animation happens? maybe it will be better to just get rid of one of them

Reply:
It is random. You can look at the audio player on the scene_init scene to see the current settings.
I figured it'd give it more variety. Though, maybe only using PageTurnBack sfx with some pitch variation would be sufficient.

There's also that, initially, I didn't think about there being a turn back animation. So we could separate the sfx to their respective transitions as well

Reply:
I'd say it should be one sound per animation type or one for all instead of random. I'm leaning slightly towards one per animation type but I don't have any strong feeling for any of these two options.

Reply:
Separated them out to each animation. I like the "turn back" sfx as a next sfx, so I reversed who the sfx are assigned to intentionally